app/models/import_service_config.rb

```diff
@@ -0,0 +1,220 @@
+# base module for all ImportServiceConfig entities
+module ImportServiceConfig
+  extend ActiveSupport::Concern
+  include ActiveModel::Model
+  include Loggable
+
+  # allow word and space characters, plus the following: - . / ( ) , :
+  ATTRIBUTE_SANITIZER = %r{[^\w\s\-./()+,:]}
+
+  CONTENT_TYPES_BY_EXT = {
+    'tar' => 'application/x-gtar',
+    'h5' => 'application/x-hdf',
+    'h5ad' => 'application/x-hdf'
+  }.freeze
+
+  ALLOWED_SCP_MODELS = [Study, StudyFile].freeze
+
+  attr_accessor :client, :user_id, :file_id, :study_id, :branding_group_id
+
+  # name of importing service (e.g. NeMO, Hca)
```

Comment:
For clarity, this seems like it should be either the conventional name of the upstream service (NeMO, HCA), or the class name of the service (Nemo, Hca), but not a mix.

```suggestion
  # name of importing service (e.g. Nemo, Hca)
```

Reply:
I agree - this has more to do with how Ruby maps class names to file names.  If we went with `NeMO`, the name of the file would actually be `ne_mo.rb`.  Since this is only used for logging, I think it's fine just just have a hard-coded return for `service_name` that maintains the correct formatting.